model/infinispan/src/main/java/org/keycloak/infinispan/compatibility/CachingCompatibilityMetadataProvider.java

```diff
@@ -31,15 +31,17 @@ private static Map<String, String> remoteInfinispanMetadata() {
         return Map.of(
                 "mode", "remote",
                 "persistence", Boolean.toString(MultiSiteUtils.isPersistentSessionsEnabled()),
-                "version", Version.getVersion()
+                "version", Version.getVersion(),
+                "jgroupsVersion", org.jgroups.Version.printVersion()
```

Comment:
Thinking about this, we don't need to store the Infinispan or the JGroups version. 
We don't have any clustering between Keycloak instances.

Reply:
done in next commit